src/tool_getparam.c

```diff
@@ -493,6 +499,32 @@ static ParameterError GetSizeParameter(struct GlobalConfig *global,
   return PARAM_OK;
 }
 
+#ifdef HAVE_UNISTD_H
+static bool CheckFile(char *fname, bool overwrite_file_prohibit)
+{
+  /* Check i the arguments are valid */
+  if(fname == NULL) {
+    printf("nullpointer warning with overwrite");
+    return TRUE;
+  }
+  /* Check if overwrite is allowed */
+  if(overwrite_file_prohibit) {
+    /* Check if file exists */
+    if(access(fname, F_OK) != -1)
+      return TRUE;
+  }
+  return FALSE;
+}
+#endif
+#ifndef HAVE_UNISTD_H
+static bool CheckFile(char *fname, bool overwrite_file_prohibit)
+{
+  /* Make this function useless if unistd is not available */
+  printf("overwrite parameter is being ignored because unistd.h is missing\n");
+  return TRUE;
+}
+#endif
+
```

Comment:
we can't do printfs from these functions, it would need a warnf global I think. but a better idea is use `stat` or something else when `access` is not available

Reply:
Ok will use a `warnf`

Reply:
stat still needs unistd.h I think